sdk/ml/azure-ai-ml/azure/ai/ml/entities/_component/_additional_includes.py

```diff
@@ -0,0 +1,402 @@
+# ---------------------------------------------------------
```

Comment:
So this is a copy from `InternalAdditionalIncludes`? Then what's the difference?

Reply:
should `InternalAdditionalIncludes` inherit this class?

Reply:
It copies lots of code logic in InternalAdditionalIncludes, but there're some differences.
1. we don't have ```.additional_include``` file in v2
2. we won't upload snapshot in current yaml path when node code in v2, but it will in internalcomponent.

The two above will result in some code diff between internal component and v2 command component, you can refer to code in new additional_include